APIKeySetup.py

```diff
@@ -0,0 +1,149 @@
+import requests
+import os
+from appJar import gui
+
+app = gui("Regulations-BOINC")
+
+
+#This code builds a window to display an error message.
+#The window can be shown by calling: app.showSubWindow("errorWindow")
+app.startSubWindow("errorWindow", "Error")
+
+app.top = True
+app.resizable = False
+app.font = {'size': 18, 'family': 'Gill Sans'}
+
+app.padding = (10, 8)
+app.guiPadding = (10, 30)
+
+app.addLabel("errorCode", "We weren't able to connect to regulations.gov.")
+app.addLabel("errorMessage", "Please try again later.")
+
+def exit(buttonName):
+    app.hideSubWindow("errorWindow")
+
+app.addButton("   Okay   ", exit)
+
+app.stopSubWindow()
+#Done building window.
+
+
+
+# This code builds a window to display an invalid API key message.
+#The window can be shown by calling: app.showSubWindow("invalidKeyWindow")
+app.startSubWindow("invalidKeyWindow", "Error")
+
+app.top = True
+app.resizable = False
+app.font = {'size': 18, 'family': 'Gill Sans'}
+
+app.padding = (50, 2)
+
+app.addLabel("errorLabel1", "Invalid API Key!")
+app.addLabel("errorLabel2", "Please visit:")
+app.link("regulations.gov", "https://regulationsgov.github.io/developers/")
```

Comment:
We visit data.gov and get a key. Then contact regulations .gov to activate the key

PLUS: Type in your link `regulations*.*gov`